RetailCoder.VBE/UI/Command/FindAllReferencesCommand.cs

```diff
@@ -163,6 +173,31 @@ private Declaration FindTarget(object parameter)
             return _state.FindSelectedDeclaration(_vbe.ActiveCodePane);
         }
 
+        private Declaration FindFormDesignerTarget()
```

Comment:
How about making an overload taking a `QualifiedModuleName` and changing this parameterless version to call the overload with the 'QualifiedModuleName` of the selected component?